README.md

```diff
@@ -5,77 +5,87 @@
-ask the community for help with this. The app would need to be available
-(initial, basic version) on 1st of July, ideally a few days earlier.
-
 Features:
 
-* Users can sign in through GitHub Oauth
-* They can create and update teams
-* Teams have members (students, coaches, mentors aka project maintainers), GitHub repositories, a log URL
-
+* Users can sign in through GitHub OAuth
+* Students can create teams and apply for the program
+* Teams can organize their members (students, coaches, mentors aka project maintainers)
 * RSS feeds are fetched from all teams' logs regularly and aggregated
-* There is a webhook endpoint for GitHub events that aggregates information about issues, pull requests and such
+* Organizers can process new projects for submission, review new teams for selection, keep track of information on conferences for students
+
+We're really excited at how much this app has grown and developed, and are really
+grateful for all of the community help along the way. There's still plenty to do
+here - and we'd love to have your contributions! Hop on over to the
+[issues](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues)
+and have a look 👀.
 
 Requirements:
 
-* Keep it simple, so Rails Girls students can get involved, too
+* Keep it simple to lower the barrier for RGSoC students to contribute too
 * By contributing, you agree to adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)
 * Make sure to check our [Contribution Guide](CONTRIBUTING.md)
 
-## System Requirements
+## Getting Started
+
+In order to contribute, you need to run the Teams App locally on your computer. For setting it up, you'll need to have some tools installed on your system:
 
 * PostgreSQL 9.5 or newer
 * Ruby 2.5.1
 * [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/)
 
-### Setup on Ubuntu
+The following section will guide you step by step through the setup and installation process on Linux 🐧 and macOS 🍏
+
+### 1. Installation
+
+#### Setup on Ubuntu 🐧
 ```bash
 # Install required packages
-$ sudo apt-get install postgresql libpq-dev libcurl3 libcurl3-gnutls libcurl4-openssl-dev postgresql-contrib-9.5 chromium-chromedriver
+sudo apt-get install postgresql libpq-dev libcurl3 libcurl3-gnutls libcurl4-openssl-dev postgresql-contrib-9.5 chromium-chromedriver
+
 # Create database user rgsoc with password rgsoc
-$ sudo -u postgres createuser -P -s rgsoc
-Enter password for new role: rgsoc
-Enter it again: rgsoc
+sudo -u postgres createuser -P -s rgsoc
+# Enter password for new role: rgsoc
+# Enter it again: rgsoc
 ```
 
-### Setup on macOS
+#### Setup on macOS 🍏
 ```bash
 # Install required packages
-$ brew install ruby postgres chromedriver
+brew install ruby postgres
 # Make sure to follow the instructions printed on the screen for postgres
-$ gem install bundler
+
+# Install chromedriver
+brew tap caskroom/cask
+brew cask install chromedriver
+
+# Install bundler
+gem install bundler
```

Comment:
I noticed we're missing this step on the 🐧 instructions. Should we add it?

I'm sure this isn't a problem for people experienced in Rails, but I can see it stumbling beginners.

Reply:
Oh, thanx for pointing out 🙏